src/app/components/help/help.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-help',
+  templateUrl: './help.component.html',
+  styleUrls: ['./help.component.scss']
+})
+export class HelpComponent implements OnInit {
+    thirdpartylicenses = '';
+    
+    constructor(
+        private httpClient: HttpClient
+    ) {}
+
+    ngOnInit() {
+        this.httpClient.get('3rdpartylicenses.txt', {responseType: 'text'})
```

Comment:
I think it may require server location

Reply:
We may also not include 3rdpartylicenses.txt in the repo, but load if available.